corehq/ex-submodules/couchforms/exceptions.py

```diff
@@ -1,4 +1,16 @@
-from couchforms.const import MAGIC_PROPERTY
+from couchforms.const import (
+    MAGIC_PROPERTY,
+    SUPPORTED_MEDIA_FILE_EXTENSIONS,
+)
+
+
+InvalidSubmissionFileExtensionErrorMessage = 'If you use multipart/form-data, please use xml file only for ' \
+                                             'submitting form xml. You may also do a normal (non-multipart) ' \
+                                             'with the xml submission as the request body instead.'
+
+InvalidAttachmentFileExtensionErrorMessage = "If you use multipart/form-data, please use the following " \
```

Comment:
nit: I don't see the value of having these as module level variables. Each is only used in a single place.

Also mixed use of `"` and `'` between the two strings.

Reply:
they were also used in tests, though I could write test differently
https://github.com/dimagi/commcare-hq/pull/28458/commits/3f5dc7992fc654f9c45e8ae9b161ffd24e5e2795
but this looks better and more inline with how the message that already existed is defined.

I also updated the messages to have "\n" in the end like it was done in the message that already existed for missing file. I think that is intentional to have spacing in case the message is displayed anywhere.